sunpy/visualization/drawing.py

```diff
@@ -194,6 +194,54 @@ def prime_meridian(axes, *, rsun: u.m = R_sun, resolution=500, **kwargs):
     return visible, hidden
 
 
+@u.quantity_input
+def extent(axes, wcs, *, rsun: u.m = R_sun, **kwargs):
```

Comment:
Why do we need rsun here? I get that draw_vertices uses it, but why do we need it here? What does it *dooo*?

Reply:
It's used to determine visibility in `_plot_vertices`. @ayshih is going to love this...

https://github.com/sunpy/sunpy/blob/4842174abcdbbb7fd2e634475329ce048cec37d3/sunpy/visualization/drawing.py#L218-L219

Reply:
Presumably this should be refactored to just use https://docs.sunpy.org/en/stable/generated/api/sunpy.coordinates.Helioprojective.html#sunpy.coordinates.Helioprojective.is_visible?

Reply:
I'm pretty sure that the `rsun` functionality in this entire module is not working as intended...  Fixing it is beyond the scope of this PR, but I'd omit `rsun` here as an argument for `extent()` until the underlying machinery is fixed.